test_output.markdown

```diff
@@ -6,9 +6,7 @@ This is a [[wiki link]].
 
 ## Code highlighting is preserved
 
-``` sql
```

Comment:
Losing syntax highlighting of the correct language is a strong regression. Why did that happen ? Can it be restored ?

Reply:
Thx, will check this!